### PR TITLE
AC Test Fix LRAC-13089 | master

### PR DESCRIPTION
**Pull request by joshitagaki**
https://issues.liferay.com/browse/LRAC-13089

### CODE REVIEW

**Comment by joshitagaki** [comment]
ci:test:sf

**Comment by joshitagaki** [comment]
I was hoping to get this test to run on CI but it has been fighting with me getting this error: https://issues.liferay.com/browse/LRAC-13083. I'm still looking into how to resolve this but it doesn't seem like it's that easy

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase [hunk 1]
```diff
@@ -642,18 +642,18 @@ definition {
 		var attributesDataTypeAscending1 = '''
 			male,agreedToTermsOfUse,defaultUser,emailAddressVerified,
 			birthday,createDate,modifiedDate,createDate,modifiedDate,status,classNameId,classPK,contactId,
-			employeeStatusId,parentContactId,prefixListTypeId,suffixListTypeId,companyId,companyId,contactId,facebookId,
-			ldapServerId,portraitId,timeZoneId,userId,emailAddress,employeeNumber,facebookSn,hoursOfOperation,
-			jabberSn,jobClass,skypeSn,smsSn,twitterSn,userName,comments,emailAddress,externalReferenceCode,firstName,
-			firstName,googleUserId,greeting,jobTitle,languageId,lastName,lastName,middleName,middleName,openId
+			employeeStatusId,parentContactId,prefixListTypeId,suffixListTypeId,companyId,contactId,facebookId,
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase:645]: Hey @joshitagaki, I think that after `classPK` it should be `companyId` and then `contactId`. Could you also check and update the list?

image

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase [hunk 2]
```diff
@@ -642,18 +642,18 @@ definition {
...
+			firstName,googleUserId,greeting,jobTitle,languageId,lastName,middleName,openId
 		''';
 		var attributesDataTypeAscending2 = "screenName,uuid";
 		var attributesDataTypeDescending1 = '''
-			emailAddress,employeeNumber,facebookSn,hoursOfOperation,
-			jabberSn,jobClass,skypeSn,smsSn,twitterSn,userName,comments,emailAddress,externalReferenceCode,firstName,
-			firstName,googleUserId,greeting,jobTitle,languageId,lastName,lastName,middleName,middleName,openId,screenName,uuid,
+			emailAddress,employeeNumber,facebookSn,firstName,hoursOfOperation,
+			jabberSn,jobClass,jobTitle,skypeSn,smsSn,twitterSn,userName,comments,emailAddress,externalReferenceCode,
+			firstName,googleUserId,greeting,jobTitle,languageId,lastName,middleName,openId,screenName,uuid,
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase:654]: Hey @joshitagaki, I think that after `jobTitle` it should be `lastName` and after `middleName` and then `skypeSn`. Could you also check and update the list?

image

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase [hunk 3]
```diff
@@ -642,18 +642,18 @@ definition {
...
 			classNameId,classPK,contactId,employeeStatusId,parentContactId,prefixListTypeId,suffixListTypeId,
-			companyId,companyId,contactId,facebookId,ldapServerId,portraitId,timeZoneId,userId,status,birthday,createDate,
+			companyId,contactId,facebookId,ldapServerId,portraitId,timeZoneId,userId,status,birthday,createDate,
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase:656]: Hey @joshitagaki, I think that after `classPK` it should be `companyId` and then `contactId`. Could you also check and update the list?

image

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase (hunk 1 above)
Comment (joshitagaki) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase]: Thanks!

**Review by ViniLopes20 — APPROVED** [review]
Verry nice @joshitagaki. Thanks for working on it. LGTM 🎉

**Review by calvinctvo — APPROVED** [review]
Thanks @joshitagaki for the fix! LGTM

**Review by manuferreira — APPROVED** [review]
Thank you Josh, LGTM. :tada:

**Pull request closed without merging**